static/js/util/integration_test_helper.tsx

```diff
@@ -169,6 +169,7 @@ export default class IntegrationTestHelper {
       const wrapper = mount(
         <Provider store={store}>
           <ReduxQueryProvider queriesSelector={getQueries}>
+            {/* @ts-ignore */}
             <Router history={this.browserHistory}>
```

Comment:
This `ignore` was not previously necessary because previously `react-router` was using `v5` history types. But that was incorrect, since it really uses v4 internally. And react-router using v5 history types broke some typings elsewhere.